docs/io/fits/appendix/faq.rst

```diff
@@ -329,17 +313,19 @@ it, because it doesn't happen until the data portion of the HDU is accessed
 (to allow things like updating the header without rescaling the data).  For
 example::
 
-    >>> hdul = fits.open('scaled.fits')
-    >>> image = hdul['SCI', 1]
+    >>> fits_scaledimage_filename = fits.util.get_testdata_filepath('scale.fits')
```

Comment:
Shall we maybe include this on data.astropy.org to avoid having get_testdata_filepath in the example?

Reply:
I don't think that's less "ugly" though. I'm open for suggestions, the input+output was (one of the) most annoying things here.